docs/en/week08/08-3.md

```diff
@@ -212,15 +212,15 @@ For DAE as we mentioned before, the sampling was happening between <span style="
 First, we encode from input space (left) to latent space (right), through encoder and noise. Next, we decode from latent space (right) to output space (left). To go from the latent to input space (the generative process) we will need to either learn the distribution (of the latent code) or enforce some structure. In our case, VAE enforces some structure to the latent space.
 
 As usual, to train VAE, we minimize a loss function. The loss function is therefore composed of a reconstruction term as well as a regularization term.
-- The reconstruction term is on the final layer (left side of the figure). This corresponds to $C(\boldsymbol{y}, \tilde{\boldsymbol{y}})$ in the figure.
-- The regularization term is on the latent layer, to enforce some specific Gaussian structure on the latent space (right side of the figure). We do so by using a penalty term $D_{KL}(\mathcal{N}(\boldsymbol{\mu}, \boldsymbol{v}) \mathrel{\Vert} \mathcal{N}(\boldsymbol{0}, \boldsymbol{1}))$. Without this term, VAE will act like a basic autoencoder, which may lead to overfitting, and we won't have the generative properties that we desire.
+- The reconstruction term is on the final layer (left side of the figure). This corresponds to $\red{C}(\vy, \vytilde)$ in the figure.
+- The regularization term is on the latent layer, to enforce some specific Gaussian structure on the latent space (right side of the figure). We do so by using a penalty term $D_{KL}(\mathcal{N}(\boldsymbol{\mu}, \vv) \mathrel{\Vert} \mathcal{N}(\boldsymbol{0}, \boldsymbol{1}))$. Without this term, VAE will act like a basic autoencoder, which may lead to overfitting, and we won't have the generative properties that we desire.
```

Comment:
`\mu` and `v` should be green.
`\vv` was used for the value in the transformer, so it's not green.